understanding/link-purpose-link-only.html

```diff
@@ -58,166 +56,113 @@
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
                   
-                  <li>This Success Criterion helps people with motion impairment by letting them skip Web
-                     pages that they are not interested in, avoiding the keystrokes needed to visit the
-                     referenced content and then return to the current content.
+                  <li>この達成基準により、運動障害のある利用者が、リンク先のコンテンツへ行って、また元へ戻ってくるという無駄なキーストロークなしに、関心のないウェブページをスキップすることができるようになる。
                      
                   </li>
                   
                   
-                  <li>People with cognitive limitations will not become disoriented by extra navigation
-                     to and from content they are not interested in.
-                     
+                  <li>認知に制約のある利用者が、余計なナビゲーションによって、必要のないコンテンツへ行ったり来たりして現在位置を見失う、ということがなくなる。                     
                   </li>
                   
                   
-                  <li>People with visual disabilities will benefit from not losing their place in the content
-                     when they return to the original page. The screen reader's list of links is more useful
-                     for finding information because the target of the links are described.
+                  <li>視覚障害のある利用者は、元のページに戻ってきたときに、コンテンツの中で居場所が分からなくなる、ということがなくなり恩恵を受ける。スクリーンリーダーのリンク一覧は、リンク先が具体的に記述されることで、情報を探す上でより便利になる。
                      
                   </li>
                   
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <dl>
                   
-                  <dt>Both an icon and text are included in the same link</dt>
+                  <dt>アイコンとテキストの両方が同じリンク内にある</dt>
                   
-                  <dd>An icon of a voting machine and the text "Government of Ireland's Commission of Electronic
-                     Voting" are combined to make a single link.
+                  <dd>自動投票機のアイコンと「アイルランド政府の電子選挙委員会」というテキストが、一つのリンクになっている。
                   </dd>
                   
-                  <dt>A list of book titles</dt>
+                  <dt>書籍名のリスト</dt>
                   
-                  <dd>A list of books is available in three formats: HTML, PDF, and mp3 (a recording of
-                     a person reading the book). The title of the book is followed by links to the different
-                     formats. The rendered text for each link is just the format type, but the text associated
-                     with each link includes the title as well as the format; for instance, "Gulliver's
-                     Travels, MP3."
+                  <dd>リストにある書籍が、HTML、PDF、そして mp3 (人が本を読み上げているのを録音したもの) の三つのフォーマットで利用可能である。書籍のタイトルの後に、様々なフォーマットへのリンクがある。例えば、描画されるリンクテキストにはフォーマット名しかないが、それぞれのリンクに関連付けられているテキストには、例えば「ガリバー旅行記 MP3 版」というように、書籍のタイトルとフォーマットの両方がある。
```

Comment:
最後の文、for instance は1つしかないのに「例えば」が2回登場するなど怪しさがありますね。render はここでは単に「表示」で良いような (テキストの表示なので)。こんな感じでしょうか。

```suggestion
                  <dd>リストにある書籍が、HTML、PDF、そして mp3 (人が本を読み上げているのを録音したもの) の三つのフォーマットで利用可能である。書籍のタイトルの後に、様々なフォーマットへのリンクがある。表示されたリンクテキストにはフォーマット名しかないが、それぞれのリンクに関連付けられているテキストには、例えば「ガリバー旅行記 MP3 版」というように、タイトルとフォーマットの両方が含まれている。
```